Solutions/EatonForeseer/Analytic Rules/EatonUnautorizedLogins.yaml

```diff
@@ -0,0 +1,59 @@
+id: 5a7fccb8-3ed0-44f2-8477-540af3ef4d92
+name: EatonForeseer - Unauthorized Logins
+description: |
+  'Detects Unauthorized Logins into Eaton Foreseer'
+severity: High
+requiredDataConnectors:
+  - connectorId: WindowsSecurityEvents
+    dataTypes:
+      - SecurityEvent
+queryFrequency: 15m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1078
+query: |
+  SecurityEvent
+  // Look for any events in the last 24 hours
+  | where TimeGenerated > ago(1d)
+  // Look for all events relating to user logins
+  | where EventID in (4624,4625,4648,4675,4634,4647)
+  | where AccountType == "User"
```

Comment:
might be a good idea to use =~ instead of ==

Reply:
Its a known account type that is logged by Windows 
I do not see a reason why we would need to account for case insensitivity

Is =~ more optimized than ==?

Reply:
@aprakash13 can you confirm this?

Reply:
Thank you for your feedback.
I am keeping the case sensitivity in this scenario as this is a known type.